### ISSUE TITLE
aq.loadCSV() within Observable fires repeatedly upon width changes

### ISSUE THREAD

**Issue by ericemc3** [uwdata/arquero#142]
Strange issue, i can see multiple network calls (and browser lagging) when i resize the window (opening the console for instance):

```
let csv_url = "https://static.data.gouv.fr/resources/jeux-de-donnees-de-formation/20210319-082923/base-naissances-2019.csv"
  
d3.dsv( ";", csv_url) // OK fires only once

aq.loadCSV(csv_url)  // fires multiple times as i resize the browser window
```

**Comment by jheer** [comment]
Huh? That is rather weird. Will have to investigate later.

**Comment by jheer** [comment]
Hmm... I bet it is the table view method definition getting re-evaluated when Observable’s built-in width value updates. That in turn might force a re-load of the entire Arquero library. Oof.

**Comment by ericemc3** [comment]
I have put up a demo here: https://observablehq.com/d/f515f6f4ad347ef0

**Comment by jheer** [comment]
I updated the primary [Arquero notebook](https://observablehq.com/@uwdata/arquero) on Observable to avoid using the built-in `width` value. Turns out it wasn't needed anyway! Hopefully all is well working as it should now. Thanks for catching this issue, and let me know if you still see any lingering problems.

**Comment by ericemc3** [comment]
Problem solved, thank you! And my wife says i shouldn't bother you on a sunday, i promise, i won't do it again :)
